### DISCUSSION TITLE
How tailwindcss.com docs work?

### DISCUSSION THREAD

**Discussion by babakfp** [tailwindlabs/tailwindcss#11374] [Help] [answered]
Hi

For example on this page https://tailwindcss.com/docs/display we have a table of all display utilities. How do you get these styles to show them on the docs page? I have this docs page https://babakfp.ir/docs/tailwindcss-addons/direction and I would like to do the same with my custom TailwindcCSS plugin and show the utilities on the docs page.

Thanks

**Comment by wongjn** [comment]
You could consider [deciphering  the source code for the page](https://github.com/tailwindlabs/tailwindcss.com/blob/master/src/pages/docs/display.mdx?plain=1) and adapting it for your own use case.

**Reply by babakfp** [reply, marked as answer]
I take a look at this code https://github.com/tailwindlabs/tailwindcss.com/blob/master/next.config.mjs#L398 and the `getUtilities` function gets the job done. It returns back the values that are used (and I guess generated) in/with a TailwindCSS plugin...
